pootle/apps/pootle_store/diff.py

```diff
@@ -80,13 +79,20 @@ def file_units(self):
         for unit in self.file_store.units:
             if unit.isheader():
                 continue
+            state = UNTRANSLATED
+            if unit.isobsolete():
+                state = OBSOLETE
+            elif unit.istranslated():
+                state = TRANSLATED
+            elif unit.isfuzzy():
+                state = FUZZY
             file_units[unit.getid()] = {
                 "unitid": unit.getid(),
                 "context": unit.getcontext(),
                 "locations": unit.getlocations(),
                 "source": unit.source,
                 "target": unit.target,
-                "state": unit.get_state_n(),
```

Comment:
becuase it returns different things for different formats. Changing this would be a significant change for ttk. Ill update the commit message